spec/lib/phobos/producer_spec.rb

```diff
@@ -234,7 +234,7 @@ class TestProducer1
             TestProducer1.producer.configure_kafka_client(kafka_client)
 
             subject.producer.async_publish_list([
-                                                  { payload: 'message-1', topic: 'topic-1', key: 'key-1' },
+                                                  { payload: 'message-1', topic: 'topic-1', key: 'key-1', headers: { foo: 'bar' } },
```

Comment:
We need to make sure we still have backwards-compatible tests where e.g. headers is not passed in.